### ISSUE TITLE
bump live_view version

### ISSUE THREAD

**Issue by superchris** [launchscout/live_elements#1]

**Comment by ArthurClemens** [comment]
I can't use this library because its LiveView version is out of date.

**Comment by superchris** [comment]
Just released a new version 0.2.1 that should allow you to use latest liveview

**Comment by camstuart** [comment]
Really enjoyed your elixir conf talk! was really hoping to give this a shot, but we may need another bump if possible:
```
mix deps.get
Resolving Hex dependencies...
Resolution completed in 0.124s
Because your app depends on live_elements ~> 0.1.0 which depends on phoenix_live_view ~> 0.18.15, phoenix_live_view ~> 0.18.15 is required.
So, because your app depends on phoenix_live_view ~> 0.20.2, version solving failed.
** (Mix) Hex dependency resolution failed
```
